sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Pyspark-Sample.ipynb

```diff
@@ -31,7 +31,7 @@
     "  \"spark.cosmos.accountKey\" : cosmosMasterKey,\n",
     "  \"spark.cosmos.database\" : cosmosDatabaseName,\n",
     "  \"spark.cosmos.container\" : cosmosContainerName,\n",
-    "  \"spark.cosmos.read.inferSchemaEnabled\" : \"true\"                          \n",
+    "  \"spark.cosmos.read.inferSchema.enabled\" : \"true\"                          \n",
```

Comment:
NIT - remove unnecessary whitespaces

Reply:
the whitespace is from jupyter notebook formatting by `nbstripout` (not introduced in this PR). we should keep the formatting by `nbstripout` tool.